thumbs/Makefile

```diff
@@ -1,6 +1,5 @@
-requirements.txt: Pipfile.lock
-	pipenv run pip freeze > requirements.txt
-	touch -r Pipfile.lock requirements.txt
+requirements.txt: pyproject.toml
+	pipenv run uv pip freeze > requirements.txt
```

Comment:
I know you changed this again in a later commit, but `pipenv run uv...` is one of the most cursed lines of code I have ever read.

![nope-honey-badger](https://github.com/user-attachments/assets/66567223-4603-4835-9128-fa5247dd6ccd)